sql/abilities.sql

```diff
@@ -523,7 +523,7 @@ INSERT INTO `abilities` VALUES (667,'blindside',15,99,4,60,173,0,0,94,2000,0,6,1
 -- INSERT INTO `abilities` VALUES (668,'deconstruction',22,1,1,0,300,0,0,???,2000,0,6,20.0,0,450,900,0,0,NULL);
 -- INSERT INTO `abilities` VALUES (669,'chronoshift',22,1,1,0,300,0,0,???,2000,0,6,20.0,0,450,900,0,0,NULL);
 -- INSERT INTO `abilities` VALUES (670,'zantetsuken',22,1,1,0,300,0,0,???,2000,0,6,20.0,0,450,900,0,0,NULL);
--- INSERT INTO `abilities` VALUES (671,'perfect Defense',22,1,1,0,300,0,0,???,2000,0,6,20.0,0,450,900,0,0,NULL);
+-- INSERT INTO `abilities` VALUES (671,'perfect_defense',22,1,1,0,300,0,0,152,2000,0,6,20.0,0,450,900,0,0,NULL);
```

Comment:
or perhaps the JA can trigger silently for the player (as long as it doesn't trigger bp cooldown and show any animation/message). This would automatically initiate petskill 671 for alexander